packages/react-components/src/lib/cesium-map/tools/geojson/geojson-to-primitive.ts

```diff
@@ -0,0 +1,42 @@
+import { FeatureCollection, Feature, Point } from 'geojson';
+import { Rectangle } from 'cesium';
+import { find } from 'lodash';
+import { PrimitiveCoordinates } from '../../data-sources';
+import { BboxCorner, DrawType } from '../../../models/enums';
+
+const POINTS_NUM = 2;
+
+export const geoJSONToPrimitive = (
+  type: DrawType,
+  geojson: FeatureCollection
+): PrimitiveCoordinates | never => {
+  switch (type) {
+    case DrawType.BOX: {
+      if (geojson.features.length !== POINTS_NUM)
+        throw new Error(`${type} must have 2 points`);
```

Comment:
add validation that BOTTOM_LEFT and TOP_RIGHT  can't be the same coordinate

Reply:
Good one, DONE